src/db/models/Contact.ts

```diff
@@ -0,0 +1,53 @@
+import { Model, DataTypes } from "sequelize";
+
+import sequelize from "./index";
+
+export default class Contact extends Model {
+    public id!: number;
+
+    public userId!: string;
+
+    public type!: string;
+
+    public value!: string;
+
+    public deletedAt!: Date;
+
+    public readonly createdAt!: Date;
+
+    public readonly updatedAt!: Date;
+}
+
+export enum ContactType {
+  PHONE_NUMBER = "PHONE_NUMBER",
+  WHATSAPP = "WHATSAPP",
+  WECHAT = "WECHAT",
+  SKYPE = "SKYPE"
+}
+
+Contact.init({
+  id: {
+    type: DataTypes.INTEGER,
+    autoIncrement: true,
+    primaryKey: true,
+  },
+  userId: {
+    type: DataTypes.INTEGER,
+    allowNull: false,
+  },
+  type: {
+    type: new DataTypes.ENUM("PHONE_NUMBER", "WHATSAPP", "WECHAT", "SKYPE"),
```

Comment:
We should make an util function, taking the enum value as an input and returning a list of string - so we only define the enum values once